Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
```

Comment:
Consider documenting the purpose of each task directly in the code, using comments. This will provide context to other developers who may work on this codebase in the future and help them understand the intention behind each task.